src/vivarium/framework/population.py

```diff
@@ -178,7 +178,12 @@ def get(self, index: pd.Index, query: str = "") -> pd.DataFrame:
             non_existent_columns = set(columns) - set(pop.columns)
             if non_existent_columns:
                 raise PopulationError(
-                    f"Requested column(s) {non_existent_columns} not in population table."
+                    f"Requested column(s) {non_existent_columns} not in population table. This is"
```

Comment:
Nit: for consistency with other multiline strings in the code base, put the space before the linebreak and start the next line right after the quote. E.g.
```
(
    "What is the airspeed velocity "
    "of an unladen swallow?"
)
```
vs
```
(
    "What is the airspeed velocity"
    " of an unladen swallow?"
)
```
Nit2: only the first string needs to be an f-string.  (This one is probably true in lots of places in the code base though)